state-sync/src/coordinator.rs

```diff
@@ -113,10 +112,12 @@ impl<T: ExecutorProxyTrait> StateSyncCoordinator<T> {
         let retry_timeout_val = match role {
             RoleType::FullNode => tick_interval_ms
                 .checked_add(node_config.state_sync.long_poll_timeout_ms)
-                .ok_or_else(|| format_err!("Fullnode retry timeout has overflown."))?,
-            RoleType::Validator => tick_interval_ms
-                .checked_mul(2)
-                .ok_or_else(|| format_err!("Validator retry timeout has overflown!"))?,
+                .ok_or_else(|| {
+                    Error::IntegerOverflow("Fullnode retry timeout has overflown!".into())
+                })?,
+            RoleType::Validator => tick_interval_ms.checked_mul(2).ok_or_else(|| {
+                Error::IntegerOverflow("Validator retry timeout has overflown!".into())
+            })?,
```

Comment:
Okay, we really should make some macros or something to get rid of this `checked_op()` stuff...

Reply:
I think this is something that the security folks are looking at (cc' @mimoo)